keps/sig-security/2763-ambient-capabilities/kep.yaml

```diff
@@ -0,0 +1,50 @@
+title: KEP Template
+kep-number: 2763
+authors:
+  - "@vinayakankugoyal"
+owning-sig: sig-security
+participating-sigs:
+  - sig-node
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2021-05-20
+reviewers:
+  - "@PushkarJ" # sig-security
+  - "@tallclair" # sig-auth
+  - "@mrunalp" # sig-node and CRI-O
+  - "@mikebrow" # sig-node and containerd
+  - "@SergeyKanzhelev" # sig-node and CRI API
+approvers:
+  - TBD
+prr-approvers:
+  - TBD
+see-also:
+  - TBD
+replaces:
+  - TBD
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "alpha"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "1.23"
+  beta: "TBD"
+  stable: "TBD"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: EnableAmbientCapabilities
```

Comment:
Usually we don't put `Enable` in [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) names - otherwise they would all collate under “E”.

Reply:
Updated to AmbientCapabilities.